test/support/commands.js

```diff
@@ -0,0 +1,8 @@
+Cypress.Commands.add('getByTestId', (testId) => cy.get(`[data-testid="${testId}"]`));
+Cypress.Commands.add('getByTitle', (title) => cy.get(`[title="${title}"]`));
+Cypress.Commands.add('showPreview', (token, fileId) => {
+    cy.get('#token-set').type(token);
```

Comment:
I realize these are inputs in the test harness, but they should ideally not use CSS selectors, regardless.